e2e/cypress/integration/channel_sidebar/hotkeys_spec.js

```diff
@@ -0,0 +1,113 @@
+// ***************************************************************
+
+import users from '../../fixtures/users';
+
+import {testWithConfig} from '../../support/hooks';
+
+import {getRandomInt} from '../../utils';
+
+const sysadmin = users.sysadmin;
+
+describe('Channel switching', () => {
+    testWithConfig({
+        ServiceSettings: {
+            ExperimentalChannelSidebarOrganization: 'default_on',
+        },
+    });
+
+    before(() => {
+        cy.apiLogin('user-1');
+
+        cy.visit('/');
+    });
+
+    const cmdOrCtrl = Cypress.platform === 'darwin' ? '{cmd}' : '{ctrl}';
+
+    it('should switch channels when pressing the alt + arrow hotkeys', () => {
+        // # Start with a new team
+        const teamName = `team-${getRandomInt(999999)}`;
+        cy.createNewTeam(teamName, teamName);
+
+        // * Verify that we've switched to the new team
+        cy.get('#headerTeamName').should('contain', teamName);
+
+        // # Press alt + up
+        cy.get('body').type('{alt}', {release: false}).type('{uparrow}').type('{alt}', {release: true});
+
+        // * Verify that the channel changed to the Off-Topic channel
+        cy.url().should('include', `/${teamName}/channels/off-topic`);
+        cy.get('#channelHeaderTitle').should('contain', 'Off-Topic');
+
+        // # Press alt + down
+        cy.get('body').type('{alt}', {release: false}).type('{downarrow}').type('{alt}', {release: true});
+
+        // * Verify that the channel changed to the Town Square
+        cy.url().should('include', `/${teamName}/channels/town-square`);
+        cy.get('#channelHeaderTitle').should('contain', 'Town Square');
+    });
+
+    it('should switch to unread channels when pressing the alt + shift + arrow hotkeys', () => {
+        // # Start with a new team
+        const teamName = `team-${getRandomInt(999999)}`;
+        cy.createNewTeam(teamName, teamName);
+
+        // * Verify that we've switched to the new team
+        cy.get('#headerTeamName').should('contain', teamName);
+
+        cy.getCurrentChannelId().as('townSquareId');
+
+        // # Create a new channel
+        // cy.createAndVisitNewChannel().as('testChannel');
+        cy.getCurrentTeamId().then((teamId) => {
+            cy.apiCreateChannel(teamId, 'test-channel', 'Test Channel').then((response) => {
+                expect(response.status).to.equal(201);
+
+                cy.wrap(response.body).as('testChannel');
+            });
+        });
+
+        // # Have another user post a message in the new channel
+        cy.get('@testChannel').then((testChannel) => cy.postMessageAs({sender: sysadmin, message: 'Test', channelId: testChannel.id}));
+
+        // # Press alt + shift + up
+        cy.get('body').type('{alt}{shift}', {release: false}).type('{uparrow}').type('{alt}{shift}', {release: true});
+
+        // * Verify that the channel changed to Test Channel and skipped Off Topic
+        cy.url().should('include', `/${teamName}/channels/test-channel`);
```

Comment:
This step failed on me.
![Screen Shot 2020-03-18 at 4 54 49 AM](https://user-images.githubusercontent.com/5334504/76900597-9e839380-68d4-11ea-81a2-db6b5d534ddf.png)

Reply:
Can you try this again with the latest changes from the feature branch? That logic may have been broken on the actual branch

Reply:
Test passed now.

Reply:
Test passed now.